types/screenshot-desktop/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for screenshot-desktop 1.12
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
This is wrong, should point to package project page or source repository:
https://github.com/bencevans/screenshot-desktop